src/main/java/org/broadinstitute/consent/http/util/CountryValidator.java

```diff
@@ -10,10 +10,27 @@
 import java.nio.charset.StandardCharsets;
 import java.util.HashSet;
 import java.util.Set;
+import org.broadinstitute.consent.http.models.DataAccessRequest;
 
 public class CountryValidator {
   private static final String FILEPATH = "assets/ISO-3166-countries.json";
   private Set<String> countries;
+
+  // For the list of countries banned, see
+  // https://www.ecfr.gov/current/title-28/chapter-I/part-202/subpart-F/section-202.601
+  private static final Set<String> bannedCountriesISO3166 =
+      Set.of(
+          "china",
+          "cuba",
+          "hong kong",
+          "iran, (islamic republic of)",
```

Comment:
```suggestion
          "iran (islamic republic of)",
```

Per:
https://github.com/DataBiosphere/consent/blob/b387c9afd52e05cbcd3d296ce2b2903cdb9a44e2/src/main/resources/assets/ISO-3166-countries.json#L107